cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceAssemblyOperatorTest.java

```diff
@@ -250,60 +249,60 @@ public void testNewToPendingProposalToStoppedRebalance(Vertx vertx, VertxTestCon
 
         Crds.kafkaRebalanceOperation(kubernetesClient).inNamespace(CLUSTER_NAMESPACE).create(kr);
 
-        when(mockKafkaOps.getAsync(CLUSTER_NAMESPACE, CLUSTER_NAME)).thenReturn(Future.succeededFuture(kafka));
+        when(mockKafkaOps.getAsync(CLUSTER_NAMESPACE, CLUSTER_NAME))
+            .thenReturn(Future.succeededFuture(kafka));
         mockRebalanceOperator(mockRebalanceOps, CLUSTER_NAMESPACE, RESOURCE_NAME, kubernetesClient, new Runnable() {
                 int count = 0;
 
                 @Override
                 public void run() {
                     if (++count == 4) {
-                        // after a while, apply the "stop" annotation to the resource in the PendingProposal state
+                        // After a while, apply the "stop" annotation to the resource in the PendingProposal state
```

Comment:
it's a personal choice ... I always start comments with low letter :-) otherwise you should fix in a TONS of places :-)